ckanext-hdx_theme/docs/code_quality/linter.rst

```diff
@@ -0,0 +1,60 @@
+Code Quality
+============
+
+Linter Configuration
+++++++++++++++++++++
+This section outlines how to configure the linter, Ruff, for your local development environment in PyCharm.
+
+Installing Ruff Linter
+----------------------
+
+Since we utilize CKAN within a Docker container, you'll need Ruff installed locally on your development machine.
```

Comment:
I would add more details about the reasons for which we need a global ruff executable. Something like ... "_The PyCharm Ruff plugin doesn't know how to use the ruff installation from the CKAN container. It needs a ruff executable on the docker host (the real machine). It also cannot execute ruff from a python virtual environment as there doesn't seem to be a way to activate a specific virtual env before running ruff.
Please note that since ruff is written in the rust programming language you don't need to necessarily install it as a python package, although you can. Please see below the various options._"